src/net/volatilevoid/nationalroaming/NationalRoaming.java

```diff
@@ -7,13 +7,31 @@
 import de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam;
 
 import java.lang.reflect.*;
+import java.util.HashMap;
 import android.telephony.ServiceState;
 import android.os.Build;
 
 public class NationalRoaming implements IXposedHookLoadPackage {
 
     private static final String TAG = "National Roaming: ";
 
+    // based on http://www.insys-icom.com/mcc/
+    public static final HashMap<String, String> baseCountryCodes = new HashMap<String, String>();

```

Comment:
Two things here:

1. You may want to lower the access level of this field. I've noted it's only used in this class, so bumping it to `private` would be best practice.
2. If you want to keep it a `public static final` field you should match naming conventions: BASE_COUNTRY_CODES.